Source/AssetRipper.Processing/AnimationClips/AnimationClipConverter.cs

```diff
@@ -44,120 +44,133 @@ private AnimationClipConverter(IAnimationClip clip, PathChecksumCache checksumCa
 
 		public static void Process(IAnimationClip clip, PathChecksumCache checksumCache)
 		{
-			AnimationClipConverter converter = new AnimationClipConverter(clip, checksumCache);
-			converter.ProcessInner();
+			//don't create instance if not needed
```

Comment:
`AnimationClipConverter` is a struct, so there's no instance actually created. It's just a local variable on the stack. Please change this back.

However, `CustomCurveResolver` is currently a class, which would allocate on the heap. It should be switched to a struct. I took a look at the implementation, and it should be as simple as changing the type declaration.

Reply:
implemented